app/src/components/Modals/delete-file-modal.tsx

```diff
@@ -100,7 +101,7 @@ const DeleteFileModal: FC<DeleteFileModalProps> = ({
                 fontWeight="semibold"
                 fontSize="button.md"
                 type="button"
-                onClick={() => alert(userData.id)}
+                onClick={() => alert(userData.userId)}
```

Comment:
Let's remove this alert and either do a log statement (using `logger`) that it's unimplemented or implement the account deletion.